apps/climatemappedafrica/src/components/HURUmap/CategoryHeader/index.js

```diff
@@ -1,31 +1,53 @@
-import { RichTypography } from "@commons-ui/legacy";
-import { Typography } from "@mui/material";
+import { Box, Typography } from "@mui/material";
 import PropTypes from "prop-types";
 import React from "react";
 
-import useStyles from "./useStyles";
-
-import Image from "@/climatemappedafrica/components/Image";
-
 function CategoryHeader({ title, description, icon, ...props }) {
-  const classes = useStyles();
-
   if (!title) {
     return null;
   }
   return (
-    <div className={classes.root}>
-      <Typography {...props} variant="h3" className={classes.title}>
+    <div>
```

Comment:
Why not use Box instead

Reply:
@koechkevin A `<Box/>` is rendered as a <`div>`. Given that we do not need to style it, is there need to use it?